tests/datasets/test_sessions_processing.py

```diff
@@ -60,3 +63,59 @@ def query_runner(
     sessions.get_default_entity().get_query_pipeline_builder().build_execution_pipeline(
         request, query_runner
     ).execute()
+
+
+granularity_tests = [
+    ({}, None),
+    ({"selected_columns": ["bucketed_started"], "groupby": ["bucketed_started"]}, 3600),
+    (
+        {
+            "selected_columns": ["bucketed_started"],
+            "groupby": ["bucketed_started"],
+            "organization": 1,
+            "project": [1],
+            "granularity": 60,
+        },
+        3600,
+    ),
+]
+
+selector_tests = [
+    (
```

Comment:
nit: please use pytest.param and provide the `id` attribute. That prints the name of the succeeding/failing tests instead of dumping the query.
Example: https://github.com/getsentry/snuba/blob/master/tests/query/test_query_ast.py#L184-L216